custom-routes-vis/src/main/webapp/script.js

```diff
@@ -23,15 +23,21 @@ const MarkerNames = {
   DESTINATION: 'destination',
 };
 
-// Array holding origin and destination markers.
-let originDestinationMarkers = []
-
-/** Creates the world map with markers. */
-function createWorldMapWithMarkers() {
-  map = new google.maps.Map(
-    document.getElementById('map'),
-    {center: {lat: 38.46049, lng: -5.428423}, zoom: 3});
-  
+let originDestinationMarkers = [];
+let displayedRoutes = [];
+const routeColors = ['#ff0000', '#eb8f1e', '#3299d1'];
```

Comment:
It might be more useful to use an enum. You can then name the colors according to what they represent. Makes the code more readable.

Reply:
Also if it is a const, use const case, so ROUTE_COLORS.

Reply:
It's easier this way to pick color according to the index of the route (see line 261)